src/internal/modules/billing/services/transactions-csv.js

```diff
@@ -13,17 +13,26 @@ const getAbsStartAndEnd = absPeriod => ({
   'Abstraction period end date': moment().month(absPeriod.endMonth - 1).date(absPeriod.endDay).format('D MMM')
 });
 
-const getAgreementsString = agreements => agreements.map(agreement => agreement.code).join(', ');
+const findAgreementValue = (agreements, code) => {
+  const value = (agreements || []).filter(agreement => agreement.code.indexOf(code) > -1);
+  return value && value.length > 0 ? `S${value[0].code}` : null;
```

Comment:
Where you return ``S${value[0].code}``, you can return ``${value[0].code}``

This is because the 'S' is part of the agreement code already, so currently this will produce `SS130x`